src/CMakeLists.txt

```diff
@@ -24,7 +24,8 @@
 set(EXE_NAME curl)
 add_definitions(-DBUILDING_CURL)
 
-if(USE_MANUAL)
+if(ENABLE_CURL_MANUAL AND HAVE_MANUAL_TOOLS)
+  add_definitions("-DUSE_MANUAL")
```

Comment:
Notice this is the only valid case we should set this C-level macro. (Missing this was causing the initial CI failures with builds having `BUILD_DOCS=ON` + `ENABLE_MANUAL=OFF`.